TEI/ParlaMint-schemaSpecs.editing.odd.xml

```diff
@@ -3785,32 +3785,62 @@
          <pm:attPreserve ident="ana"/>
       </attList>
       <exemplum xmlns:xi="http://www.w3.org/2001/XInclude" xml:lang="en">
-         <egXML xmlns="http://www.tei-c.org/ns/Examples" source="#TEI">
-            <affiliation>Junior project officer for the US <name type="org">National Endowment for
-        the Humanities</name>
-            </affiliation>
-         </egXML>
-      </exemplum>
-      <exemplum xmlns:xi="http://www.w3.org/2001/XInclude"
-                xml:lang="en"
-                versionDate="2018-10-30">
-         <p>This example indicates that the person was affiliated with the
-      Australian Journalists Association at some point between the dates
-      listed.</p>
-         <egXML xmlns="http://www.tei-c.org/ns/Examples" source="#TEI">
-            <affiliation notAfter="1960-01-01" notBefore="1957-02-28">Paid up member of the
-        <orgName>Australian Journalists Association</orgName>
-            </affiliation>
-         </egXML>
-      </exemplum>
-      <exemplum xmlns:xi="http://www.w3.org/2001/XInclude"
-                xml:lang="en"
-                versionDate="2018-10-30">
-         <p>This example indicates that the person was affiliated with Mount Holyoke College throughout the entire span of the date range listed.</p>
-         <egXML xmlns="http://www.tei-c.org/ns/Examples" source="#TEI">
-            <affiliation from="1902-01-01" to="1906-01-01">Was an assistant professor at Mount Holyoke College.</affiliation>
+        <egXML xmlns="http://www.tei-c.org/ns/Examples" source="#ParlaMint">
+	   <person xml:id="PeterFowler">
+                  <persName>
+                     <forename>Peter</forename>
+                     <surname>Fowler</surname>
+                  </persName>
+                  <affiliation from="1970-06-18" ref="#parla.lower" role="MP" to="1974-02-28"/>
+                  <affiliation from="1970-06-18"
+                               ref="#party.CON"
+                               role="member"
+                               to="2001-06-07"/>
+                  <affiliation from="1974-02-28" ref="#parla.lower" role="MP" to="1974-10-10"/>
+                  <affiliation from="1974-10-10" ref="#parla.lower" role="MP" to="1979-05-03"/>
+                  <affiliation from="1979-05-03" ref="#parla.lower" role="MP" to="1983-06-09"/>
```

Comment:
@katjameden. 
`ref` should point to `<org>`, `parla.lower` is an element in taxonomi:
https://github.com/clarin-eric/ParlaMint/blob/bc6257eeb657cf62ec74a1a37731b294deef7c20/Data/ParlaMint-SI/ParlaMint-SI.xml#L189-L196

Can you extend affiliation examples with the `ana` attribute. `ana` should point to `org/event` in cases of government and parliament.

Reply:
@matyaskopp, first comment: it might not be a good idea to mege documentation-desc with documentation already, as the changes now sometimes affect many elements, and can also turn out to be less than ideal, and, once this is noted, could be reverted or changed in another way. Let's wait until we have a coherent set of changes before merging.

Second comment, to your comment 

> ref should point to org, parla.lower is an element in taxonomy

to have the example it as it is now (so, wrong) was a concious decision, i.e. we decided to put in as examples samples from ParlaMint, as it is *now*: the example above comes from [ParlaMint-GB.xml](https://github.com/clarin-eric/ParlaMint/Data/ParlaMint-GB/ParlaMint-GB.xml).

The reason is that we don't yet know all the changes that we will make and how, so it seems better to put in the examples the current state, and, once this is fixed in the XML files, also change the guidelines. Otherwise, we might wind up in a (even more) messy situation, I think.

However, if you think the final decision has already been made, feel free to change the pertinent example!

Reply:
> @matyaskopp, first comment: it might not be a good idea to mege documentation-desc with documentation already, as the changes now sometimes affect many elements, and can also turn out to be less than ideal, and, once this is noted, could be reverted or changed in another way. Let's wait until we have a coherent set of changes before merging.

I don't want to merge it. I just needed to know what was going on there. You made big changes and the easiest way to see the differences is to create a draft pull request.

>The reason is that we don't yet know all the changes that we will make and how, so it seems better to put in the examples the current state, and, once this is fixed in the XML files, also change the guidelines. Otherwise, we might wind up in a (even more) messy situation, I think.
> However, if you think the final decision has already been made, feel free to change the pertinent example!

I think that the decision is final. 

But I am not sure if it is possible to fix simply GB data as discussed here https://github.com/clarin-eric/ParlaMint/issues/191#issuecomment-1074052043. 

GB should be fixed in this way:
- split parliament into two organizations
  -  lower house `HouseOfCommons` - with events corresponding to terms
  - upper house `HouseOfLords` with no terms events  
- every affiliation `#parla.lower` should be changed to  `#HouseOfCommons`, and the corresponding event should be added to `@ana`
- every affiliation `#parla.upper` should be changed to  `#HouseOfLords`

But GB affiliate all MPs to `#parla.lower`:
```XML
          <person xml:id="SarahMullally">
             <persName>
                <forename>Sarah</forename>
                <surname>Mullally</surname>
             </persName>
             <sex value="F">Female</sex>
             <affiliation from="2018-04-10" ref="#parla.lower" role="MP"/>
             <affiliation from="2018-04-10" ref="#party.BI" role="member"/>
             <idno subtype="contact" type="URI">https://members.parliament.uk/member/4696/contact</idno>
          </person>
```

But _Sarah Mullally_ is a member of the House of Lords according to wiki: https://en.wikipedia.org/wiki/Sarah_Mullally

Reply:
I have updated comment in related issue: https://github.com/clarin-eric/ParlaMint/issues/177#issue-1158204464
- CZ case is now fixed (affiliate with organization and `@ana` contains corresponding event)
- GB affiliates with `<category>` which is definitely wrong according to https://www.tei-c.org/release/doc/tei-p5-doc/en/html/ref-affiliation.html
- IS and IT wrongly encodes only ministers - affiliates persons with government events
- BG affiliates with URL outside of corpus
- other issues are that there are no affiliations (missing `@ref`)